### PR TITLE
Add Bongiovi’s Digital Power Station

### PR DESCRIPTION
**Pull request by Josh-Tilles**

### CODE REVIEW

File: Casks/digital-power-station.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url 'http://www.dpsplugin.com/download/BongioviAcousticsDPS.dmg'
+  name 'Digital Power Station'
+  name 'DPS Plugin'
+  homepage 'http://dpsplugin.com/home/'
+  license :closed
+
+  pkg 'Digital Power Station Installer.pkg'
+
+  uninstall :pkgutil => 'com.bongiovi.pkg.DigitalPowerStation.*'
+  # TODO should prefer `/Applications/Digital Power Station/Uninstaller.app` instead?
```
Comment (vitorgalvao) [inline Casks/digital-power-station.rb]: We don’t support graphical installers (and don’t intend on it).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ff20cd0a9898d1b3b8d9e2ff3234cdcd5fc0412b. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/digital-power-station.rb (hunk 1 above)
Comment (MerelyAPseudonym) [inline Casks/digital-power-station.rb]: Good to know.

**Comment by MerelyAPseudonym** [comment]
@vitorgalvao thank you for the explanation. Everything looks good!

**Pull request closed without merging**